publisher/scheduler.py

```
@@ -0,0 +1,55 @@
+from django_q.tasks import schedule
+
+def schedule_task(planned_post):
+    """
+    Ставит задачу через django-q на указанное время.
+    Возвращает объект Schedule (или None).
+    """
+
+    schedule = schedule(
```

Ruff (0.12.2)

Local variable `schedule` referenced before assignment

(F823)